dump-event-schema-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/model/TypeDefinition.scala

```diff
@@ -0,0 +1,8 @@
+package org.virtuslab.akkasaferserializer.model
+
+final case class TypeDefinition(
+    typeSymbol: String,
+    name: String,
+    annotations: Seq[String],
```

Comment:
Not sure if addressed yet... `annotations` would be better off optional in the resulting JSON, since in 90-99% of cases (depending on the serializer) this will be empty, annotations aren't common in Scala

Reply:
It is optional already, it is handled here:
https://github.com/VirtusLab/akka-safer-serializer/blob/6856608db929c9b4d0e71be1dbc0ce562df2b479/sbt-dump-event-schema/src/main/scala/org/virtuslab/akkasaferserializer/DumpEventSchema.scala#L20

Reply:
Making field an `Option[A]` does not make the output more concise. Making it more concise requires a more forceful approach.

Reply:
Okay, sorry, didn't notice